flyctl/app_config_test.go

```diff
@@ -54,3 +54,21 @@ func TestLoadTOMLAppConfigWithServices(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, p.Definition, rawData)
 }
+
+func TestLoadTOMLAppConfigWithEnvVars(t *testing.T) {
+	path := "./testdata/env-vars.toml"
+	p, err := LoadAppConfig(path)
+	assert.NoError(t, err)
```

Comment:
This should probably be `require.NoError(t, err)` as the rest of the test is pointless unless the configuration is loaded.

Reply:
Oh, does assert not fail immediately? Will change.